tests/integration/security/rbac/v2/group/group_test.go

```diff
@@ -174,11 +169,12 @@ func TestGroupV2RBAC(t *testing.T) {
 			time.Sleep(60 * time.Second)
 
 			for _, tc := range cases {
-				testName := fmt.Sprintf("%s->%s:%s/%s",
+				testName := fmt.Sprintf("%s->%s:%s%s[%v]",
```

Comment:
"%s%s" in the Sprintf makes it difficult to distinguish two variables printed. Suggest adding a delimiter between them.

Reply:
There is already a `/` in the path, adding another `/` makes it hard to find out the real path.

Reply:
IIUC, the "%s%s" in the Sprintf is for tc.Request.Options.PortName and tc.Request.Options.Path. If without a delimiter between them, they can be "name1path1", which makes it hard to tell which is for port name and which is for path.

Reply:
Like I said, the path already includes the `/` character. 
Without this change, it prints like the following:
```
--- PASS: TestRBACV2 (76.29s)
    --- PASS: TestRBACV2/b->a:http//xyz (0.66s)
    --- PASS: TestRBACV2/b->a:tcp/ (0.40s)
    --- PASS: TestRBACV2/c->a:http// (0.67s)
    --- PASS: TestRBACV2/c->a:tcp/ (0.31s)
    --- PASS: TestRBACV2/d->a:http// (0.72s)
    --- PASS: TestRBACV2/d->a:tcp/ (0.41s)
    --- PASS: TestRBACV2/a->b:http//xyz (0.67s)
    --- PASS: TestRBACV2/a->b:http//secret (0.28s)
    --- PASS: TestRBACV2/a->b:tcp/ (0.34s)
    --- PASS: TestRBACV2/c->b:http// (0.42s)
    --- PASS: TestRBACV2/c->b:tcp/ (0.38s)
    --- PASS: TestRBACV2/d->b:http// (0.30s)
    --- PASS: TestRBACV2/d->b:tcp/ (0.40s)
    --- PASS: TestRBACV2/a->c:http// (0.39s)
    --- PASS: TestRBACV2/a->c:http//secrets/admin (0.33s)
    --- PASS: TestRBACV2/a->c:tcp/ (0.38s)
    --- PASS: TestRBACV2/b->c:http// (0.40s)
    --- PASS: TestRBACV2/b->c:http//credentials/admin (0.29s)
    --- PASS: TestRBACV2/b->c:tcp/ (0.30s)
    --- PASS: TestRBACV2/d->c:http// (0.40s)
    --- PASS: TestRBACV2/d->c:http//any_path/admin (0.30s)
    --- PASS: TestRBACV2/d->c:tcp/ (0.33s)
    --- PASS: TestRBACV2/a->d:http//xyz (0.28s)
    --- PASS: TestRBACV2/a->d:tcp/ (0.32s)
    --- PASS: TestRBACV2/b->d:http// (0.40s)
    --- PASS: TestRBACV2/b->d:tcp/ (0.39s)
    --- PASS: TestRBACV2/c->d:http//any_path (0.37s)
    --- PASS: TestRBACV2/c->d:tcp/ (0.30s)
PASS
```

With this change, it prints like the following:
```
--- PASS: TestRBACV2 (91.22s)
    --- PASS: TestRBACV2/b->a:http/xyz[false] (0.79s)
    --- PASS: TestRBACV2/b->a:tcp[false] (0.46s)
    --- PASS: TestRBACV2/c->a:http/[false] (0.91s)
    --- PASS: TestRBACV2/c->a:tcp[false] (0.52s)
    --- PASS: TestRBACV2/d->a:http/[false] (0.88s)
    --- PASS: TestRBACV2/d->a:tcp[false] (0.37s)
    --- PASS: TestRBACV2/a->b:http/xyz[false] (0.92s)
    --- PASS: TestRBACV2/a->b:http/secret[false] (0.38s)
    --- PASS: TestRBACV2/a->b:tcp[false] (0.47s)
    --- PASS: TestRBACV2/c->b:http/[false] (0.42s)
    --- PASS: TestRBACV2/c->b:tcp[false] (0.40s)
    --- PASS: TestRBACV2/d->b:http/[false] (0.43s)
    --- PASS: TestRBACV2/d->b:tcp[false] (0.35s)
    --- PASS: TestRBACV2/a->c:http/[false] (0.41s)
    --- PASS: TestRBACV2/a->c:http/secrets/admin[false] (0.35s)
    --- PASS: TestRBACV2/a->c:tcp[false] (0.34s)
    --- PASS: TestRBACV2/b->c:http/[false] (0.42s)
    --- PASS: TestRBACV2/b->c:http/credentials/admin[false] (0.40s)
    --- PASS: TestRBACV2/b->c:tcp[false] (0.40s)
    --- PASS: TestRBACV2/d->c:http/[false] (0.39s)
    --- PASS: TestRBACV2/d->c:http/any_path/admin[false] (0.36s)
    --- PASS: TestRBACV2/d->c:tcp[false] (0.46s)
    --- PASS: TestRBACV2/a->d:http/xyz[true] (0.38s)
    --- PASS: TestRBACV2/a->d:tcp[false] (0.40s)
    --- PASS: TestRBACV2/b->d:http/[true] (0.42s)
    --- PASS: TestRBACV2/b->d:tcp[false] (0.40s)
    --- PASS: TestRBACV2/c->d:http/any_path[true] (0.39s)
    --- PASS: TestRBACV2/c->d:tcp[false] (0.40s)
PASS
```

Reply:
1. The delimiter does not have to be "/", e.g., the delimiter can be ":", "#", "[]" or any other string that separate two variables printed.
2. With the code in this PR, an example output is "PASS: TestRBACV2/b->a:http/xyz[false] (0.79s)", in which "http/xyz" is for tc.Request.Options.PortName and tc.Request.Options.Path. But it is difficult to figure out which part of "http/xyz" is for PortName and which part is for Path. An example way to make it easier to read can be: fmt.Sprintf("%s->%s: [%s], [%s], [%v]", ...), which uses "[]," to delimit the variables to print.

Reply:
I think Yangmin's approach is cleaner. fmt.Sprintf("%s->%s: [%s], [%s], [%v]", ...) is too verbose and not very intuitive to look at. It's clear to me that tcp and http are port name (or protocol) and the path after / is the path (and tcp does not have path).